src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/Scripts/AzureDiskEncryptionPreRequisiteSetup.ps1

```diff
@@ -62,10 +62,11 @@ $ErrorActionPreference = "Stop"
         $defaultHomePage = 'http://contoso.com';
         $now = [System.DateTime]::Now;
         $oneYearFromNow = $now.AddYears(1);
-        $aadClientSecret = [Guid]::NewGuid();
+        $aadClientSecret = [Guid]::NewGuid().ToString();
```

Comment:
tests?  Please ensure this is tested end-to-end, and either add covering test automation as part of thisPR, or add a test debt issue in the repo to add test automation for the January release and link the issue here.

Reply:
Hey Mark, this is a sample / example usage script, which was manually validated to be working fine. I sign off based on the manual vaidation

Reply:
OK, can we add some validation so that we know when this script fails in the future, or would that be ifficult

Reply:
Opened issue https://github.com/Azure/azure-powershell/issues/5081  to track this